src/utils/xdrTypeFromValue.js

```diff
@@ -8,6 +9,9 @@ export function xdrTypeFromValue (xdrEnum, value) {
   try {
     return xdrEnum.values().filter(item => item.value === value)[0]
   } catch (error) {
-    throw new Error(`xdrEnumToConstant: Cannot get values from provided xdrEnum (${xdrEnum})`)
+    throw new Error(globalize('xdr-type-from-value.error', {
```

Comment:
leave the previous version. This place does not localize

Reply:
done